### PR TITLE
Update cheatsheet for 24.04 release

### PR DESCRIPTION
**Pull request by carkod**
## Done
Update cheat sheet according to [copy docs](https://docs.google.com/document/d/1uI5hsuhe5VUpFSSiQas6FVB2F7RhlbNXcr5Pvr_Lm2c/edit)

## QA
- Go to [homepage](https://multipass-run-325.demos.haus/), scroll down to the cheat sheet section 
- Check that [copy doc comments](https://docs.google.com/document/d/1uI5hsuhe5VUpFSSiQas6FVB2F7RhlbNXcr5Pvr_Lm2c/edit) are addressed

## Issue
Fixes https://warthogs.atlassian.net/browse/WD-10168

### CODE REVIEW

**Comment by akbarkz** [comment]
@carkod Can you please specify which ticket corresponds to this change in the description and the title of this PR?
Also, please change the text of the button in the form to "Download Cheat Sheet" instead of current "Download the pro tips"

File: templates/index.html [hunk 1]
```diff
@@ -99,7 +99,7 @@ <h2>Designed for developer convenience</h2>
 <section class="p-strip">
   <div class="row u-equal-height u-vertically-center">
     <div class="col-5 u-align--center u-hide--small u-hide--medium">
-      <img src="https://assets.ubuntu.com/v1/33d66ec7-Image-catalogue.svg" style="height: 171px; width: 300px;" alt=""/>
+      <img src="https://assets.ubuntu.com/v1/639c1f12-server-cli-cheatsheet.png" style="height: 171px; width: 300px;" alt=""/>
```
Comment (akbarkz) [inline templates/index.html:102]: For some reason the image is not updated in the demo

Comment (akbarkz) [reply templates/index.html:102]: now it's showing nothing

**Review by akbarkz — APPROVED** [review]

**Comment by carkod** [comment]
To be merged ~ 18th of April

**Pull request merged**